### PR TITLE
update routes and ridership link

### PR DESCRIPTION
**Pull request by gregbunce**
point to their new arcgis enterprise hosted map

### CODE REVIEW

File: src/pages/products/sgid/transportation/uta-routes.astro [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const metadata: IMetadata = {
 
 const page: IPageMetadata = {
   ...metadata,
-  application: 'https://rideuta.maps.arcgis.com/apps/webappviewer/index.html?id=448c3cfe44264f1e8931073dfb6e5417',
+  application: 'https://maps.rideuta.com/portal/apps/instant/sidebar/index.html?appid=b5db22dc09a24203886607cf3e2abb30',
```
Comment (steveoh) [inline src/pages/products/sgid/transportation/uta-routes.astro:30]: **question**: did you also update this in the sgid index?

**Review by steveoh — APPROVED** [review]
once the question is resolved we're good to go

File: src/pages/products/sgid/transportation/uta-routes.astro (hunk 1 above)
Comment (gregbunce) [reply src/pages/products/sgid/transportation/uta-routes.astro:30]: i did now. ha. thanks for the reminder.

Comment (steveoh) [reply src/pages/products/sgid/transportation/uta-routes.astro:30]: i could use some help keeping the sgid index current and useful.

**Pull request merged**